Change the exportToPDF Overview demo description (#1773)
DevExpress/devextreme-demos: JSDemos/Demos/DataGrid/PDFOverview/description.md

> The [DataGrid](/Documentation/ApiReference/UI_Components/dxDataGrid/) allows you to export its contents to a PDF document. To enable PDF export operations, import the <a href="https://github.com/parallax/jsPDF" target="_blank">jsPDF</a> library.

Reviewer: I think the second sentence deserves a separate paragraph. The first sentence introduces the entire idea of the demo - export the grid to PDF (what it's all about). The second is one of the steps to get there (something you should do).

> Call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method that belongs to the [pdfExporter](/Documentation/ApiReference/Common/Utils/pdfExporter/) module to export DataGrid content to a PDF document. Specify at least two required properties:

Reviewer: I don't want to repeat the part that says "to export DataGrid content to a PDF document" because the first sentence in this topic already says that and uses almost all the same words. So this reads repetitive. And that phrase complicates the sentences (long sentences that has too many words "to").

```suggestion
Call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method that belongs to the [pdfExporter](/Documentation/ApiReference/Common/Utils/pdfExporter/) module. Specify at least two required properties:
```

- [jsPDFDocument](/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/#jsPDFDocument)    
Specifies the **jsPDF** instance.

- [component](/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/#component)    
Specifies the DataGrid's instance.

> In this demo, click the **Export to PDF** button to call the **exportDataGrid(options)** method in the button's [onClick](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#onClick) handler.

Reviewer: Let's remove the hyperlink. The point of the sentence is to draw the reader's attention to the code in this demo. We should not encourage them to go and read the description of the onClick event. Maybe even this version is better: 

```suggestion
In this demo, click the **Export to PDF** button to initiate export. To review implementation, see the **exportDataGrid(options)** method call in the button's **onClick** handler.
```

The [DataGrid](/Documentation/ApiReference/UI_Components/dxDataGrid/) allows you to export its contents to a PDF document. 

To enable PDF export operations, import the <a href="https://github.com/parallax/jsPDF" target="_blank">jsPDF</a> library.

Call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/pdfExporter/#exportDataGridoptions) method that belongs to the [pdfExporter](/Documentation/ApiReference/Common/Utils/pdfExporter/) module. Specify at least two required properties:

- [jsPDFDocument](/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/#jsPDFDocument)    
Specifies the **jsPDF** instance.

- [component](/Documentation/ApiReference/Common/Object_Structures/PdfExportDataGridProps/#component)    
Specifies the DataGrid's instance.

> In this demo, click the **Export to PDF** button to initiate export. To review implementation, see the **exportDataGrid(options)** method call in the button's **onClick** handler.

Reviewer 2: ```suggestion
In this demo, click the **Export to PDF** button to initiate export. To review implementation details, see the **exportDataGrid(options)** method call in the button's **onClick** handler.
```
